specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerConnectivityConfiguration.json

```diff
@@ -0,0 +1,362 @@
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfigurationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsList": {
+            "$ref": "./examples/NetworkManagerConnectivityConfigurationList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ConnectivityConfigurationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectivityConfiguration"
+          },
+          "description": "Gets a page of Connectivity Configurations"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next page of results."
+        }
+      },
+      "description": "Result of the request to list network manager connectivity configurations. It contains a list of configurations and a link to get the next set of results."
+    },
+    "ConnectivityConfiguration": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ConnectivityConfigurationProperties",
+          "description": "Properties of a network manager connectivity configuration"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The network manager connectivity configuration resource"
+    },
+    "ConnectivityConfigurationProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "A friendly name for the resource."
+        },
+        "description": {
+          "type": "string",
+          "description": "A description of the connectivity configuration."
+        },
+        "connectivityTopology": {
+          "type": "string",
+          "enum": [
+            "HubAndSpokeTopology",
+            "MeshTopology"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectivityTopology",
+            "modelAsString": true
+          },
+          "description": "Connectivity topology type."
+        },
+        "hubId": {
+          "type": "string",
+          "description": "The hub vnet Id."
+        },
+        "isGlobal": {
+          "type": "boolean",
+          "description": "Flag if global mesh is supported."
+        },
+        "appliesToGroups": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/connectivityGroupItem"
+          },
+          "description": "Groups for configuration"
+        },
+        "provisioningState": {
```

Comment:
I see provisioningState defined for resources. Are PUTs long running operations? Is your RP following pattern listed here: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put
If yes, please add below...
        "x-ms-long-running-operation": true,
        "x-ms-long-running-operation-options": {
          "final-state-via": "original-uri"
        },

For more details, http://azure.github.io/autorest/extensions/#x-ms-long-running-operation

Reply:
This comment applicable for all resources with provisioningState declared operations are long running.

Reply:
As per our conversation, I understand the PUT is not long running operation.

---
In reply to: [574213894](https://github.com/Azure/azure-rest-api-specs/pull/12441#discussion_r574213894) [](ancestors = 574213894)

Reply:
It is not long running op, and we use provisioningState  to record the operation succeed or not